www/pages/docs/css-and-images.md

```diff
@@ -105,4 +105,21 @@ customElements.define('x-header', HeaderComponent);
 > }
 >
 > customElements.define('x-header', HeaderComponent);
-> ```
\ No newline at end of file
+> ```
+
+
+### Meta files
+
+The build includes a few [default copy plugins](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/cli/src/plugins/copy) for common meta files.   Typically, they support copying the files from the source directory to the build directory.
+
+* `plugin-copy-favicon.js` - copies favicon.ico.  NOTE:  Does not support multiple favicons.  For multiple support, use the manifest.json or make a custom plugin
```

Comment:
Nice, I think I will tweak this as it interweaves public API / private implementation / development + testing details and so I think for the sure of readers, probably just listing out the supported files are is good.  Though you make a good callout that if you need _more_ support, then linking to the copy plugin is a good idea, so will definitely keep that in.

As a side note, _manifest.json_ is more related to Greenwood's own build output, so not really something for users per se, but I do have a PR that is documenting more of that as we speak, as it pertains to [adapter plugins](https://github.com/ProjectEvergreen/greenwood/pull/1237/files#diff-adb56e4ca10fb02f0c5a8695f668985a028ad038a5bab81bd85359167770e638R37) anyway, and will likely get its own "Build Output" section when migrated to the new docs website soon.